### DISCUSSION TITLE
mDISH feature in Hepatic Explorer

### DISCUSSION THREAD

**Discussion by JimBuchanan** [SafetyGraphics/safetyGraphics#486] [Chart Discussion]
# Name of the chart
mDISH

# Objectives 
How to identify possible Hy's Law cases when ALT and/or bilirubin values are abnormal at baseline.

# User Interface mockup

![mDISH](https://user-images.githubusercontent.com/38111134/106031708-6b796700-6084-11eb-98e7-0c5e7224ca4d.JPG)

# Features
The fold change in ALT and bilirubin is calculated based on the "baseline" ALT and bilirubin values. "Baseline" can be defined in one of two ways: 1) the defined baseline lab sample, or 2) the mean of the screening value and the Day 1 pre-dose value. The simplest programmatically would be the single baseline value.

# Data Requirement
Only the laboratory data domain is required. For the single lab value method, the field would either be AVISIT="Baseline" or VISIT="Screening"

# References
Aithal GP, Watkins PB, Andrade RJ. Case definition and phenotype standardization
in drug-induced liver injury. Clin Pharmacol Ther. 2011;89:806-815.

Lin X, Parks D, Painter J, et al. Validation of multivariate outlier detection analyses used to identify potential drug-induced liver injury in clinical trial populations. Drug Safety 2012;35:865-875.

**Comment by jwildfire** [comment]
Nice summary, @JimBuchanan - certainly seems doable. Are you thinking this would be a static display? Or are you looking to modify the mDish implementation in hep-explorer from the screenshot above?

**Reply by JimBuchanan** [reply]
I'm looking at this as a modification of the mDISH implementation. Looking ahead, I can foresee the drop down choices of display type can have eDISH, mDISH, fold-change from a predefined baseline, nR modified Hy's Law and, ultimately, Palt defined Hy's Law.
